internal/provisioner/model/names.go

```diff
@@ -82,17 +82,33 @@ func (c *Contour) EnvoyRBACNames() RBACNames {
 // ComponentLabels returns labels for a Contour component.
 func (c *Contour) ComponentLabels() map[string]string {
```

Comment:
I'd suggest a rename to `AppLabels`, along with an update to the godoc indicating that this is specifically for the Deployment/DaemonSet, to help differentiate from the below func.

Reply:
done